vue2-vuetify/src/complex/OneOfRenderer.vue

```diff
@@ -111,22 +118,26 @@ const controlRenderer = defineComponent({
     const input = useJsonFormsOneOfControl(props);
     const control = (input.control as any).value as typeof input.control;
 
-    const selectedIndex = ref(control.indexOfFittingSchema || 0);
-    const tabIndex = ref(selectedIndex.value);
+    const selectedIndex = ref(control.indexOfFittingSchema);
```

Comment:
because is always selects the first one and in case where we have a dropdown now and the property is required we need explicitly to be selected by the end user - otherwise the dropdown will be automatically selected to the first one